cpm

```diff
@@ -107,85 +107,85 @@ has() {
 
 _apt() {
     case "$OP" in
-        install) apt install "$@";;
-        remove) apt remove "$@";;
+        install) sudo apt install "$@";;
+        remove) sudo apt remove "$@";;
         list) apt list --installed;;
         count) dpkg-query -f '.\n' -W | tot;;
-        update) apt update;;
-        upgrade) apt dist-upgrade;;
+        update) sudo apt update;;
+        upgrade) sudo apt dist-upgrade;;
         search) apt search "$1";;
         show) apt show "$1";;
-        clean) apt autoremove;;
+        clean) sudo apt autoremove;;
     esac
 }
 
 _dnf() {
     case "$OP" in
-        install) dnf install "$@";;
-        remove) dnf remove "$@";;
+        install) sudo dnf install "$@";;
+        remove) sudo dnf remove "$@";;
         list) dnf list --installed;;
         count) rpm -qa | tot;;
-        update) dnf check-update;;
-        upgrade) dnf distro-sync;;
+        update) sudo dnf check-update;;
+        upgrade) sudo dnf distro-sync;;
         search) dnf search "$1";;
         show) dnf info "$1";;
-        clean) dnf autoremove;;
+        clean) sudo dnf autoremove;;
     esac
 }
 
 _nix() {
     case "$OP" in
-        install) nix-env -iA "$@";;
-        remove) niv-env -e "$@";;
+        install) sudo nix-env -iA "$@";;
+        remove) sudo niv-env -e "$@";;
         list) nix-env -q "$@";;
         count) nix-env -q | wc -l;;
-        update) nix-channel --update;;
-        upgrade) nix-env -u;;
+        update) sudo nix-channel --update;;
+        upgrade) sudo nix-env -u;;
         search) nix-env -qa "$@";;
         show) nix-env -qa --description "$@";;
-        clean) nix-collect-garbage -d;;
+        clean) sudo nix-collect-garbage -d;;
```

Comment:
You wouldn't want to use sudo with nix. It's a user level package manager, so this actually changes meaning.

Reply:
Cool! I'll do that, I don't really use Nix lol